x-pack/plugins/ml/server/models/fields_service/fields_service.ts

```diff
@@ -35,14 +38,28 @@ export function fieldsServiceProvider({ asCurrentUser }: IScopedClusterClient) {
    */
   async function getAggregatableFields(
     index: string | string[],
-    fieldNames: string[]
+    fieldNames: string[],
+    datafeedConfig?: DatafeedOverride
   ): Promise<string[]> {
     const { body } = await asCurrentUser.fieldCaps({
       index,
       fields: fieldNames,
     });
     const aggregatableFields: string[] = [];
+    const datafeedAggConfig = datafeedConfig?.aggregations ?? datafeedConfig?.aggs;
```

Comment:
nit, `datafeedAggregations` or even just `aggregations` might be a better name for this as `datafeedAggConfig` is a bit too similar to `datafeedConfig`. I missed the difference when first reading it.
The same in other files in this PR.

Reply:
Updated here https://github.com/elastic/kibana/pull/81923/commits/f54c8e8dc5d0419538d1270cb91407de4f45bd42